Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityInputSystem.cs

```diff
@@ -62,6 +62,14 @@ public interface IMixedRealityInputSystem : IMixedRealityEventSystem
         /// </summary>
         bool IsInputEnabled { get; }
 
+        /// <summary>
```

Comment:
If you want to do this in a way that isn't breaking, consider spawning a different interface:

IMixedRealityCapabilityChecker

Which would be implementable by other systems like spatial awareness (and I guess then you'd probably need a static helper/other thing that would federate calling out to the other systems since this might be beyond just input)

Reply:
>  consider spawning a different interface

That would be consistent with the IMixedRealityDataProviderAccess interface. I'll make the change